patch/kconfig-inclusions

```diff
@@ -63,8 +61,6 @@ CONFIG_SENSORS_MP2888=m
 CONFIG_SENSORS_MP2975=m
 CONFIG_SENSORS_TMP421=m
 CONFIG_SENSORS_STTS751=m
-CONFIG_SENSORS_ADS1015=m
-CONFIG_SENSORS_EMC2305=m
 CONFIG_SENSORS_POWR1220=m
```

Comment:
not sure whether CONFIG_SENSORS_POWR1220  is also only for SN2201?

Reply:
The only sensors I removed were the ones that are not included in the patch set that I have uploaded.